sdk/sdktypes/build_export.go

```diff
@@ -42,3 +43,9 @@ func (r BuildExport) WithLocation(loc CodeLocation) BuildExport {
 func (r BuildExport) WithSymbol(sym Symbol) BuildExport {
 	return BuildExport{r.forceUpdate(func(pb *BuildExportPB) { pb.Symbol = sym.String() })}
 }
+
+func (r BuildExport) Location() CodeLocation {
+	return kittehs.Must1(CodeLocationFromProto(r.read().Location))
```

Comment:
Don't `panic`, return an invalid location if parsing fails.

Reply:
if you call this function, it means that the state of BuildExport must be valid. InvalidLocation would mean none, but it must never error.